packages/client/mutations/handlers/handleAuthenticationRedirect.ts

```diff
@@ -14,7 +14,13 @@ const handleAuthenticationRedirect: OnNextHandler<
   SendClientSegmentEventMutation(atmosphere, 'User Login')
   const redirectTo = getValidRedirectParam()
   if (redirectTo) {
-    history.push(redirectTo)
+    const nonce = new URLSearchParams(window.location.search).get('nonce')
+    const {authToken} = atmosphere
+    if (nonce) {
```

Comment:
That's not the best criteria, but right now we don't pass the `'admin-login'` page type to the mutation